index.html

```diff
@@ -0,0 +1,193 @@
+  <!-- <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'> -->
+</head>
+<body>
+  <div class="container">
+    <header class="app-header">
+        <img src="images/My Logo.png" alt="Portifolio logo" class="my_logo">
+        <img src="images/Icon - Menu.png" alt="Portifolio logo" class="menu_bar">
+    </header>
+    <section class="section_1"><br>
+      <h2 class="greatting">I’m Eric.<br>
+          Glad to see you!
+      </h2>
+        <p>I'm software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <div class="social_media1">
+        <h2 class="connect">LET'S CONNECT</h2>
+        <div class="social">
+          <a href=""><img src="images/Vector2.png" alt=""></a>
+          <a><img src="images/Linkedin icon.png" alt=""></a>
+          <a><img src="images/Vector3.jpg" alt=""></a>
+          <a><img src="images/Vector5.png" alt=""></a>
+          <a><img src="images/Vector4.png" alt=""></a>
+        </div>
+      </div>
+    </section>
+    <section class="section_2">
+      <div class="tonic_image_1">
+        <img src="images/tonic_image1.jpg" alt="Tonic image">
+      </div>
+      <div>
+      <div class="tonic_div">
+        <h2>Tonic</h2>
+        <ul>
+          <li class="tonic_title">CANOPY</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">Back End Dev</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">2015</li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="section_2">
+      <div class="tonic_image_1">
+        <img src="images/post_story1.png" alt="Tonic image">
+      </div>
+      <div>
+      <div class="tonic_div">
+        <h2>Multi-Post
+          Stories</h2>
+        <ul>
+          <li class="tonic_title">CANOPY</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">Back End Dev</li>
+          <li><img src="images/Counter.png" alt=""></li>
+          <li class="tonic_content">2015</li>
+        </ul>
+      </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="features">
+          <li><button>HTML</button></li>
+          <li><button>CSS</button></li>
+          <li><button>JavaScript</button></li>
+        </ul>
+        <div class="view_button">
+          <button>See Project</button>
```

Comment:
- [ ] Please make sure your `see project button` has the sudo state of pressed implemented.
![states](https://user-images.githubusercontent.com/99656687/236603923-e2e79049-0846-4a26-bed1-dbd0acc4663a.png)